x/sockopt/is_sending_bytes_not_implemented.go

```diff
@@ -0,0 +1,17 @@
+//go:build !linux
```

Comment:
maybe it's worth noting that this is not even needed on some platforms (we know of mac/ios already)

Reply:
Should be handy for Mac/iOS for disorder strategy. But it needs a separate implementation, based on `tcp_connection_info` struct: https://developer.apple.com/documentation/kernel/tcp_connection_info

Reply:
I haven't had split issues on macOS, so we may not need it.